externs/olx.js

```diff
@@ -3538,7 +3538,8 @@ olx.interaction.PointerOptions.prototype.handleUpEvent;
  *     features: (ol.Collection.<ol.Feature>|undefined),
  *     filter: (ol.SelectFilterFunction|undefined),
  *     wrapX: (boolean|undefined),
- *     hitTolerance: (number|undefined)}}
+ *     hitTolerance: (number|undefined),
+ *     zIndex: (number|undefined)}}
  */
```

Comment:
You'll also need to add a block to document the `zIndex` option here.  See the [`hitTolerance` option](https://github.com/bvx89/openlayers/blob/c4fee4551a838f176fbd671c1c22d6fcb2f64f6d/externs/olx.js#L3656-L3663) for an example.